src/repositories/tagsRepository.js

```diff
@@ -22,3 +22,13 @@ export const getTagByName = async (tagName) => {
   );
   return tag[0];
 };
+
+export const getTags = async () => {
+  const { rows: tag } = await connection.query(`
```

Comment:
because tag is array with multiple elements, it looks nicer when the variable name is in plural.

Reply:
Done.